packages/gatsby/src/redux/reducers/schema.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionsUnion, IGatsbyState } from "../types"
+
+export const schemaReducer = (
+  state: IGatsbyState["schema"] | {} = {},
```

Comment:
hey, instead of unioning this to an empty object, can we just have the default initialization be an instance of graphql schema. It's dangerous to not have this be the expected shape in the rest of the codebase.

Reply:
Sure! I updated the code to be:
```ts
state: IGatsbyState["schema"] = new GraphQLSchema({ query: null }),
```

Is it the solution you described or should I add a query within it? If yes, I might need more guidance for that 😸